src/components/link-list/config/index.js

```diff
@@ -0,0 +1,33 @@
+export const args = {
```

Comment:
Als de objecten maar op 1 plek worden gebruikt hoeven ze van mij niet in een aparte folder, maar is personal preference

Reply:
Ik vind het persoonlijk niet mis om dat consistent op deze manier te fixen. Dan is het overal mooi duidelijk. Als je wil wil ik het er nog wel uithalen.

Reply:
Akkoord, maar dan moeten we misschien voor in de toekomst eens nadenken of de naam `config` wel juist is